server/views/pages/licence/PSS.test.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs'
+import cheerio from 'cheerio'
+import nunjucks, { Template } from 'nunjucks'
+import { registerNunjucks } from '../../../utils/nunjucksSetup'
+
+const snippet = fs.readFileSync('server/views/pages/licence/PSS.njk')
+
+describe('Print a PSS licence', () => {
+  let compiledTemplate: Template
+  let viewContext: Record<string, unknown>
+  const njkEnv = registerNunjucks()
+
+  beforeEach(() => {
+    compiledTemplate = nunjucks.compile(snippet.toString(), njkEnv)
+    viewContext = {}
+  })
+
+  it('verify render of a PSS licence', () => {
```

Comment:
Generally I'd split out this many `expect()`s into separate tests with specific test goals/descriptions, fine for now though

Reply:
These will definitely evolve to be more detailed, and different, tests of AP, PSS and AP_PSS licences. Each will have their own specifics.